api/ceramic.yaml

```diff
@@ -293,10 +293,22 @@ paths:
           required: false
         - name: limit
           in: query
-          description: the maximum number of events to return, default is 10000.
+          description: The maximum number of events to return, default is 100. The max with data is 10000.
           required: false
           schema:
             type: integer
+        - name: includeData
+          in: query
+          required: false
+          schema:
+            type: string
+            enum: [none, payload, full]
+          description: >
+            Whether to include the event data (carfile) in the response.
+            In the future, only the payload or other options may be supported:
+              * `none` - Empty, only the event ID is returned
+              * `payload` - Only the signed event payload data as a carfile. Unsigned and time events are unchanged.
```

Comment:
probably what we actually want for time events here is literally just the timestamp and prev and nothing else.  In the same way that the agg library will assume ceramic-one is doing signature validation, it can also assume that ceramic-one has verified the time event, and so the only actual information that the time event conveys up to the consumer is the new timestamp.

Reply:
hmm, I was trying to stick to the protocol types, is there something that exists that conveys that? we have the event and the proof, but it seems like you're suggesting part of both in something new or am I misunderstanding? I'm definitely open to returning something else if it makes sense.

```rust
pub struct RawTimeEvent {
    id: Cid,
    prev: Cid,
    proof: Cid,
    path: String,
}

pub struct Proof {
    chain_id: String,
    root: Cid,
    tx_hash: Cid,
    tx_type: String,
}
```

Reply:
yeah it would be something new.  Problem is that right now ceramic-one doesn't even have the timestamp anywhere since it isn't stored in the event data itself, it comes from looking up the tx_hash against the actual blockchain.  So we won't be able to return the timestamp until ceramic-one is doing event validation for time events.  But maybe we could still put the structure in place and just return a placeholder timestamp for now?

Reply:
I'm willing to do that but would appreciate if @PaulLeCam would chime in. 

Maybe it's worth chatting about event validation? These payloads don't have to stick directly to the protocol for consumers and we can provide a higher level construct that is useful. For now, the signed event payload is just implied as validated, and this feels like it would be a `validated::TimeEvent`. Since we have the `full` option, anyone is free to do it all themselves if they don't like what we provide via the `payload` parameter.

Reply:
As Spencer wrote I think it would be useful eventually to have the timestamp provided for TimeEvents, one thing it needs to keep having though is the `id` to identify the stream it belongs to.

I agree though that it would be good to define what event validation will provide and possibly start defining the higher-level structures that ceramic-one could expose.

Reply:
Going to defer this conversation to the conclusion API. For now I removed the payload option and we only return the complete event envelope if requested.